ui/.env.development

```diff
@@ -14,4 +14,13 @@ REACT_APP_USER_DOCS_URL=[{"href":"https://github.com/gojek/turing","label":"Turi
 # Link to the .proto file used by the Kafka Result Logger when using Protobuf serialization
 REACT_APP_RESULT_LOG_PROTO_URL=
 
+# Default Experiment Engine Remote UI settings
+REACT_APP_DEFAULT_EXPERIMENT_ENGINE_REMOTE_APPNAME=
+REACT_APP_DEFAULT_EXPERIMENT_ENGINE_REMOTE_URL=
+
 REACT_APP_OAUTH_CLIENT_ID=*** set clientId ***
+
+# JSON string map, key being the (unroutable) path used by the remote app for API calls,
+# and value being the URL to proxy to. Used only in local development mode, to bypass CORS.
+# More info: https://create-react-app.dev/docs/proxying-api-requests-in-development/
+REMOTE_COMPONENTS_PROXY_PATHS=
```

Comment:
Used by `setupProxy.js` for local development. `REACT_APP_` prefix is not required here.